utilities/ps.py

```diff
@@ -27,20 +30,19 @@ def print(self):
 
 def get_ps() -> list[str]:
     """
-    Invoke Docker "ps" with a filter to get the configuration for all services. The
-    info is returned as a list of plain text lines.
+    Invoke Docker Compose "ps" with the JSON format flag and
+    return a list of lines of JSON output.
 
     :return: the output printed by the subprocess to STDOUT
     """
 
     proc = subprocess.run(
         [
-            "docker",
+            "just",
+            "dc",
```

Comment:
This creates a dependency on `just` that's not present in the original script. I don't mind, because `just` must be present to invoke the `ps` recipe anyway.
```suggestion
            "docker",
            "compose",
```

Reply:
In what meaningful circumstance would `just` not be present?

If you don't use `just dc` you'll also need to pass the compose file and setup the environment arguments that `just` sets. Directly using `docker compose` with our stack don't work so well, in my experience.

If there's a real situation in which the `ps` script is used and `just` wouldn't be present, then sure, but I can't think of one :shrug: We rely on `just` for the most basic operations of Openverse development and it's a hard, non-negotiable requirement for working in our codebase, on the level of having system Python and bash available.

Reply:
I'm going to merge with the two approvals now, but @dhruvkb feel free to open a follow-up issue/PR if you think avoiding `just` is a must for this script.